lxpanel-plugin/kano_updater.c

```diff
@@ -8,7 +8,8 @@
 
 #include <gtk/gtk.h>
 #include <gdk/gdk.h>
-#include <glib/gi18n.h>
+#define GETTEXT_PACKAGE "kano-updater"
+#include <glib/gi18n-lib.h>
```

Comment:
@hzulla Sorry, I'm not too familiar with gettext, do we need to compile this file with all the languages or is this just providing a link to the languages that we've defined?